### DISCUSSION TITLE
Display change on print

### DISCUSSION THREAD

**Discussion by Jogroe** [dompdf/dompdf#2902] [Q&A] [answered]
When i generate my PDF file from the html everything looks like expected. But when i try to print the file, the overview of the PDF change (e.g: my table is larger than expected and exceed the page). Any ideas what i should do ? (i'm using chrome)

**Comment by bsweeney** [comment, marked as answer]
Maybe the page size needs to be adjusted to better fit the actual paper size? Could be that the PDF is being fitted to the page size and causing the content to overflow. I'd check the paper size and scale in the Chrome print dialog.

**Comment by Jogroe** [comment]
thanks for the solution !
